docs/user-guide/add-an-application.md

```diff
@@ -9,14 +9,23 @@ Below is an example of linking the existing application to your project
 
 ![Add Application](/images/user-guide/add-an-application/add-application.png)
 
-The flow for adding an existing application is different to all the commands encounterd so far. First the prompts for an application name and description are shown. Enter these and submit them when you are satisfied with your values. Subatomic will then pull a list of all the repositories in your Bitbucket project and you can select which one is holds the source for the application you wish to use. Once you have made you selection you will be informed that your application is being provisioned.
+The flow for adding an existing application is different to all the commands encounterd so far. First the prompts for an application name and description are shown. Enter these and submit them when you are satisfied with your values. Subatomic will then pull a list of all the repositories in your Bitbucket project and you can select which one holds the source for the application you wish to use. Once you have made your selection you will be informed that your application is being provisioned.
```

Comment:
I think say "application is being created" here. Provisioned in for us usually refers to the environment creation etc...